etc/profile-a-l/com.github.tchx84.Flatseal.profile

```diff
@@ -0,0 +1,57 @@
+# Firejail profile for flatseal
+# This file is overwritten after every install/update
+# Persistent local customizations
+include com.github.tchx84XXgg12uC.Flatseal.local
+# Persistent global definitions
+include globals.local
+
+# Required for GTK theem to work:
+read-only ${HOME}/.config/dconf/
+noblacklist ${HOME}/.config/dconf/
+whitelist ${HOME}/.config/dconf/
```

Comment:
The current GTK theme didn't pick up without this, but I expected it to after granting access to dconf's D-Bus interface.

Reply:
Thanks for the clarification, I'd no idea.